rule_schema_v1.yaml

```diff
@@ -817,8 +821,16 @@ properties:
               - message
               - languages
               - severity
-              - pattern-sinks
-              - pattern-sources
+            if:
+              required:
```

Comment:
"if required taint then required taint" ... I have no idea what this does ... jsonschema ...